src/test/java/org/broadinstitute/hellbender/tools/walkers/vqsr/VariantRecalibratorIntegrationTest.java

```diff
@@ -388,6 +392,65 @@ public void testVariantRecalibratorSampling() throws IOException {
         spec.executeTest("testVariantRecalibratorSampling"+  inputFile, this);
     }
 
+    // Expected exception is a UserException but gets wrapped in a RuntimeException
+    @Test(expectedExceptions = RuntimeException.class)
+    public void testVariantRecalibratorFailedRscriptOutput() throws IOException {
+        final String inputFile = getLargeVQSRTestDataDir() + "phase1.projectConsensus.chr20.1M-10M.raw.snps.vcf";
```

Comment:
The integration test spec and these arguments are a lot. I would recommend taking the `String[] VQSRSNPParamsWithResources` above and adding your rscript arguments to it and running that through the runCommandLineArguments() method. Furthermore i think this test is actually sketchy since it could be the case that R is properly installed... The second one is probably sufficient.